web/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+//profile.active = dev
```

Comment:
Why commented? How are you gonna switch between profiles

Reply:
> Why commented? How are you gonna switch between profiles

The file contains available profiles, inactive profiles are commented and 1 active profile remains. If you need to switch a profile, uncomment one profile and comment other.